browser/BUILD.gn

```diff
@@ -150,6 +150,7 @@ source_set("browser_process") {
     "//brave/components/ntp_background_images/browser",
     "//brave/components/ntp_tiles",
     "//brave/components/p3a",
+    "//brave/components/permissions",
```

Comment:
actually not sure whether our component should be named "permissions", maybe "permissions_lifetime" or something

Reply:
I see it as additional logic in the existing chromium "permissions" component. Looking at brave-core it looks like every time we extend an already existed component, the name in ```//brave/components``` stays the same.

Reply:
I don't think this is needed here since it's already in https://github.com/brave/brave-core/pull/7982/files#diff-6a608c4f57aeddf569a5dc7343eb324c723c142082d5a004812b3e80c7ba485cR10

Reply:
This PR is just a first part. There is another PR on the way with a KeyedService which will be created through a factory in ```//brave/browser/permissions/```.

Reply:
Then the dep should be added when it's actually needed. We don't want extra deps that aren't necessary because they increase build time.

Reply:
yep, you're right. removed it in this PR.